Lab/Challenge5.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lab
+{
+    public class Challenge5
+    {
+        public static int GetLargest(int[] arr)
+        {
+            int largest = 0;
```

Comment:
This should either be `arr[0]` (make sure you test for empty array!) or `int.MinValue`.